src/dos/cdrom.h

```diff
@@ -410,8 +412,37 @@ class CDROM_Interface_Image final : public CDROM_Interface
 	static int           refCount;
 };
 
+class CDROM_Interface_Physical : public CDROM_Interface {
+public:
+	~CDROM_Interface_Physical() override;
+
+	bool GetAudioStatus(bool& playing, bool& pause) override;
+	bool PlayAudioSector(const uint32_t start, uint32_t len) override;
```

Comment:
Probably better to remove `const` for primitive types (non-reference & non-pointer) in function declarations and only keep them in function definitions.

Reply:
Removed const from all primitive type declarations in this file

Reply:
Sorry, can you drop the commit as per our recent discussion?

Reply:
Commit dropped